packages/web/pages/_app.tsx

```diff
@@ -1,9 +1,13 @@
 import { ChakraProvider, CSSReset, MetaTheme } from '@metafam/ds';
 import { PageHeader } from 'components/PageHeader';
+import { CONFIG } from 'config';
 import { Web3ContextProvider } from 'contexts/Web3Context';
+import { withUrqlClient } from 'next-urql';
 import { AppProps } from 'next/app';
 import Head from 'next/head';
 
+import { getTokenFromStore } from '../lib/auth';
```

Comment:
can we ensure that in future commits we remove any relative path imports?
imo makes it cleaner.